bin/upgrade.bash

```diff
@@ -143,17 +151,22 @@ main() {
     exit 1
   fi
 
+  export CK8S_CLUSTER="${cluster}"
```

Comment:
If you go this route I would actually skip having the cluster as an argument altogether and only use an environment variable set in the shell. Don't mix environment variables and arguments.

Reply:
Sure, I kind of followed how the `CK8S_TARGET_VERSION` variable was used, as I thought it was most important to be consistent with how arguments was passed to the `check_config` function. So there might be more work that should be done for some of the other global variables used

Reply:
Alright, please have another look now in case I missed something :sweat_smile: